src/main/java/org/wso2/carbon/connector/ldap/SearchEntry.java

```diff
@@ -108,6 +136,10 @@ private OMElement prepareNode(SearchResult entityResult, OMFactory factory, OMNa
 
         for (int i = 0; i < returnAttributes.length; i++) {
             attribute = attributes.get(returnAttributes[i]);
+            if (returnAttributes[i].contains(";")) {
```

Comment:
Can we add a comment on why we are doing this?

Reply:
Done